lib/jake/array.ex

```diff
@@ -1,5 +1,49 @@
 defmodule Jake.Array do
-  def gen(_) do
-    StreamData.constant([])
+  def gen(spec) do
+    items = Map.get(spec, "items", %{})
+    uniq = Map.get(spec, "uniqueItems", false)
+    additional_items = Map.get(spec, "additionalItems", %{})
+    max_items = Map.get(spec, "maxItems", 100)
+    min_items = Map.get(spec, "minItems", 0)
+
+    case {items, uniq, additional_items} do
+      {items, true, _} when items == %{} ->
+        StreamData.string(:ascii)
+        |> StreamData.uniq_list_of(max_length: max_items, min_length: min_items)
```

Comment:
why do we use string here? do we need to the special case here?

Reply:
Forgot to mention it, It was originally Jake.gen(%{}). But it was causing infinite loop. So used string as the sensible default

Reply:
could you try to figure out what causes the infinite loop and fix the source of the problem.